NitroxClient/GameLogic/ItemContainers.cs

```diff
@@ -27,9 +27,8 @@ public ItemContainers(IPacketSender packetSender, LocalPlayer localPlayer)
         public void BroadcastItemAdd(Pickupable pickupable, Transform ownerTransform)
         {
             string ownerGuid = null;
-
             bool isCyclopsLocker = Regex.IsMatch(ownerTransform.gameObject.name, @"Locker0([0-9])StorageRoot$", RegexOptions.IgnoreCase);
-            bool isEscapePodStorage = ownerTransform.parent.name == "EscapePod";
+            bool isEscapePodStorage = ownerTransform.parent.name == "EscapePod" || ownerTransform.parent.name == "EscapePod(Clone)";
```

Comment:
You can do .StartsWith("EscapePod")

Reply:
I tried it, but then the escapePodStorage starts with that as well, so it would break things, but I'll try it again